### DISCUSSION TITLE
what is it used for

### DISCUSSION THREAD

**Discussion by 779102260** [jellydn/twt-dl-cli#215] [Q&A]
Why rely on the vxtwitter.com interface, what is it used for? Will the interface crash?
```
fetch(url.replace("twitter.com", "vxtwitter.com"), {
    headers: {
      "User-Agent": "TelegramBot (like TwitterBot)",
    },
})
```

**Comment by jellydn** [comment]
Just FYI https://www.reddit.com/r/discordapp/comments/v1axyv/what_is_vxtwitter_i_have_never_heard_anything_of/
